documentation/modules/rhv-prerequisites.adoc

```diff
@@ -11,5 +11,4 @@ The following prerequisites apply to {rhv-full} migrations:
 * You must use a xref:compatibility-guidelines_{context}[compatible version] of {rhv-full}.
 * You must have the CA certificate of the {manager}.
 +
-You can obtain the CA certificate by navigating to `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA` in a browser.
-If the {manager} Apache CA certificate was replaced with a third-party CA certificate, you must have both the {manager} Apache CA certificate and {manager} CA certificate.
+If you did not replace the {manager} Apache CA certificate with a third-party CA, you can obtain the required CA certificate by navigating to https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA in a browser. Otherwise, you need to specify the third-party certificate.
```

Comment:
@RichardHoch 
I would suggest to change (including line 12):
"You must have the CA certificate of the {manager} or the Apache CA certificate of the {manager} if the {manager} Apache CA certificate was replaced by a third-party CA certificate. 
You can obtain the CA certificate by navigating to `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA` in a browser."